code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm

```diff
@@ -295,6 +294,20 @@
 	M.update_icons()
 	M.clean_blood()
 
+/datum/reagent/lube
+	name = "Space Lube"
+	description = "Lubricant is a substance introduced between two moving surfaces to reduce the friction and wear between them. giggity."
```

Comment:
Capitalize the giggity please, it's after all after a period.

Reply:
the giggity was not my doing, I thought I removed it